src/LockToVotePlugin.sol

```diff
@@ -12,7 +12,7 @@ import {IProposal} from "@aragon/osx-commons-contracts/src/plugin/extensions/pro
 import {SafeCastUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/math/SafeCastUpgradeable.sol";
 import {MajorityVotingBase} from "./base/MajorityVotingBase.sol";
 import {ILockToGovernBase} from "./interfaces/ILockToGovernBase.sol";
-import {PluginUUPSUpgradeable} from "@aragon/osx-commons-contracts/src/plugin/PluginUUPSUpgradeable.sol";
+import {PluginUUPSUpgradeable} from "./lib/PluginUUPSUpgradeable.sol";
```

Comment:
@brickpop how can the project build when you are trying to import `PluginUUPSUpgradeable.sol` from the `lib` folder, but that file does not exist there?

Reply:
Confusion solved: you need to build using the `make` script and not by directly using `forge build`

Reply:
Also, @brickpop, I think that this import is not needed in `LockToVotePlugin`. Probably it was needed before because of `/// @inheritdoc PluginUUPSUpgradeable` (old code)

Reply:
@StErMi I went for the "hands off" approach to remove human error and to make things automatic. But it could still be added to version control, as a way to flag if this file ever changes. 

The source package should not have existed in the first place (should be just OSx), there's little chance that this file will be changed at the source